nucypher/cli/commands/deploy.py

```diff
@@ -397,6 +394,7 @@ def upgrade(general_config, actor_options, retarget, target_address, ignore_depl
         filepath = f'proposal-{trustee.multisig_agent.contract_address[:8]}-TX-{transaction_proposal.nonce}.json'
         transaction_proposal.write(filepath=filepath)
         emitter.echo(SUCCESSFUL_SAVE_MULTISIG_TX_PROPOSAL.format(filepath=filepath), color='blue', bold=True)
+        return  # Exit
```

Comment:
Is it just to emphasize the end of execution here? Doesn't seem like it's necessary.

Reply:
Haha we had a conversation about this - it was @KPrasch personal preference for a clear end of execution.

Reply:
It may be a sign that (in Justin's voice) these branches want to be functions

Reply:
That's a great @jMyles' impersonation.

Wrt to your comment, there's a tendency in the codebase to add some syntactic redundancy for the sake of clarity (e.g., redundant `returns`, wrapping boolean expressions with `bool(...)`). I personally don't think they hurt, so I'm completely OK with that.